### PR TITLE
feat: add method for fetching wellbore trajectory

### PR DESCRIPTION
**Pull request by Sollimann**
[SDL-127] - Fetch trajectory for wellbore in JS SDK

[SDL-127]: https://cognitedata.atlassian.net/browse/SDL-127

### CODE REVIEW

File: packages/wells/src/client/api/wellboresAPI.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    this.client = httpClient;
+  }
+
+  public set setProject(project: String) {
+    this.project = project;
+  }
+
+  /* eslint-disable */
+  public getTrajectory = async (wellboreId: number): Promise<Survey | undefined> => {
+      const path: string = `/${this.project}/wellbores/${wellboreId}/trajectory`
```
Comment (drsphelps) [inline packages/wells/src/client/api/wellboresAPI.ts]: should an error be returned if the project is not defined? I think the API will just return a 404, which wouldn't very helpful?

File: packages/wells/src/client/api/wellboresAPI.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      let trajectory: Survey | undefined;
+      await this.client?.get<Survey>(path)
+      .then(response => {
+        if (response.status == 200){
+          trajectory = response.data;
+        } else {
+          trajectory = undefined;
```
Comment (drsphelps) [inline packages/wells/src/client/api/wellboresAPI.ts]: here we will be hiding the error message received from the API, perhaps we could have some logging or maybe throw an error to help the user debug?

**Review by drsphelps — COMMENTED** [review]
A couple of comments, although you may have good reasons for doing it the way you are

File: packages/wells/src/client/api/wellboresAPI.ts (hunk 1 above)
Comment (Sollimann) [reply packages/wells/src/client/api/wellboresAPI.ts]: I guess we can set a 400 Bad Request if the project is not set in the client?

Comment (drsphelps) [reply packages/wells/src/client/api/wellboresAPI.ts]: Sounds good!

File: packages/wells/src/client/api/wellboresAPI.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+      if (this.project == undefined){
+        throw new HttpError(400, "The client project has not been set.", {})
+      }
+
+      const path: string = `/${this.project}/wellbores/${wellboreId}/trajectory`
+      
+      return await this.client?.get<Survey>(path)
+      .then(response => response.data)
+      .catch(err => {
```
Comment (drsphelps) [inline packages/wells/src/client/api/wellboresAPI.ts:27]: This is super clean!

**Review by drsphelps — APPROVED** [review]
LGTM!

**Pull request merged**